decidim-core/app/helpers/decidim/layout_helper.rb

```diff
@@ -85,7 +85,7 @@ def extended_navigation_bar(items, max_items: 5)
       extra_items = items.slice((max_items + 1)..-1) || []
       active_item = items.find { |item| item[:active] }
 
-      render partial: "decidim/shared/extended_navigation_bar", locals: {
+      render partial: "decidim/shared/extended_navigation_bar.html", locals: {
```

Comment:
Why do we need to add the extension? 🤔

Reply:
It's wierd, but since the partial is called within a `cell` I needed to add the extension so it could be found and all works then.

There's another PR for this landing page that needed the same, maybe there's another workaround.